src/main/java/com/github/housepower/jdbc/data/Block.java

```diff
@@ -63,6 +66,16 @@ public void setObject(int i, Object object) throws SQLException {
 
     public void setConstObject(int i, Object object) throws SQLException {
         objects[i] = object;
+        constantColumn[i] = true;
+    }
+
+    public void resetIndex() {
+        for (int i = 0; i < columns.length; i++) {
+            objects[i] = null;
+            nameWithPosition.put(columns[i].name(), i + 1);
+            columnIndexAdds[i] = i;
+            constantColumn[i] = false;
+        }
     }
 
     public void incrIndex(int i) {
```

Comment:
This may be removed.

Reply:
No, it can't.
Now in `clearBatch`, we cleared the block and re-fill the constant value. Everytime you call `setConstantObject`, it would offset the position. If you call it many times, it would offset many times. So we have to reset the index position to ensure the position is correct.